receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers.go

```diff
@@ -312,18 +313,77 @@ func getAge(start time.Time, end time.Time) int64 {
 func (ghs *githubScraper) getCVEs(
 	ctx context.Context,
 	client graphql.Client,
+	rClient *github.Client,
 	repo string,
 ) (map[metadata.AttributeCveSeverity]int64, error) {
-	c, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo)
-	if err != nil {
-		return nil, err
+	d := ghs.getDbotAlerts(ctx, client, repo)
+	c := ghs.getCodeScanAlerts(ctx, rClient, repo)
+
+	return mapSeverities(d, c), nil
+}
+
+func (ghs *githubScraper) getDbotAlerts(
+	ctx context.Context,
+	client graphql.Client,
+	repo string,
+) []CVENode {
+
+	var all []CVENode
+	var cursor *string
+
+	for hasNextPage := true; hasNextPage; {
+		alerts, err := getRepoCVEs(ctx, client, ghs.cfg.GitHubOrg, repo, cursor)
+
+		if err != nil {
+			ghs.logger.Sugar().Errorf("error %v getting dependabot alerts from repo %s", zap.Error(err), repo)
+			return nil
+		}
+
+		hasNextPage = alerts.Repository.VulnerabilityAlerts.PageInfo.HasNextPage
+		cursor = &alerts.Repository.VulnerabilityAlerts.PageInfo.EndCursor
+		all = append(all, alerts.Repository.VulnerabilityAlerts.Nodes...)
+
+	}
+
+	return all
+}
+
+// Get the Code Scanning Alerts count for a repository via the REST API
+func (ghs *githubScraper) getCodeScanAlerts(
+	ctx context.Context,
+	client *github.Client,
+	repoName string,
```

Comment:
```suggestion
	repo string,
```

Reply:
Resolved, made the change consistently across other functions